src/views/bullet_list.ts

```diff
@@ -30,6 +31,11 @@ export class BulletListView<
     if (this.contentDOM) {
       const type = (this.node.attrs.listStyleType as JatsStyleType) || 'bullet'
       this.contentDOM.style.listStyleType = JATS_HTML_LIST_STYLE_MAPPING[type]
+
+      if (this.node.attrs.dataTracked) {
+        const classes = [...getChangeClasses(this.node.attrs.dataTracked)]
```

Comment:
Do we really need to unpack the return value of getChangeClasses?

Reply:
updated